tests/PHPUnit/Unit/ConfigTest.php

```diff
@@ -390,5 +390,47 @@ public function test_forceSave_writesNothingIfThereAreNoChanges()
 
         @unlink($configFile);
     }
+    
+    public function testFromGlobalConfig()
+    {
+        $userFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/config.ini.php';
+        $globalFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/global.ini.php';
+        $commonFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/common.config.ini.php';
+        
+        $config = Config::getInstance();
```

Comment:
@diosmosis why are those are used in the other tests? https://github.com/ThaDafinser/piwik/blob/patch-3/tests/PHPUnit/Unit/ConfigTest.php#L52-L54

i updated the tests :-)